src/node/waku_node.py

```diff
@@ -58,15 +59,26 @@ def start(self, **kwargs):
             "nat": f"extip:{self._ext_ip}",
             "peer-exchange": "true",
             "discv5-discovery": "true",
+            # "cluster-id": "2",
         }
 
-        if "go-waku" in self._docker_manager.image:
+        if self.is_gowaku():
             go_waku_args = {
                 "min-relay-peers-to-publish": "1",
                 "legacy-filter": "false",
                 "log-level": "DEBUG",
             }
             default_args.update(go_waku_args)
+        elif self.is_nwaku():
+            nwaku_args = {
+                "metrics-server": "true",
+                "metrics-server-address": "0.0.0.0",
+                "metrics-server-port": self._metrics_port,
+                "metrics-logging": "true",
+            }
+            default_args.update(nwaku_args)
+        else:
+            raise NotImplemented("Not implemented for this node type")
```

Comment:
This should be `NotImplementedError`. The one you used is a Singleton used to actually return a value for cases such as comparisons.

Reply:
Good catch, fixing!